ax/models/torch/botorch_modular/sebo.py

```diff
@@ -300,26 +295,34 @@ def _optimize_with_homotopy(
                 )
             ],
         )
-        batch_initial_conditions = get_batch_initial_conditions(
-            acq_function=self.acqf,
-            raw_samples=optimizer_options_with_defaults["raw_samples"],
-            # pyre-fixme[6]: For 3rd argument expected `Tensor` but got
-            #  `Union[Tensor, Module]`.
-            X_pareto=self.acqf.X_baseline,
-            target_point=self.target_point,
-            bounds=bounds,
-            num_restarts=optimizer_options_with_defaults["num_restarts"],
-        )
+
+        if "batch_initial_conditions" not in optimizer_options:
+            optimizer_options["batch_initial_conditions"] = (
+                get_batch_initial_conditions(
+                    acq_function=self.acqf,
+                    raw_samples=optimizer_options_with_defaults["raw_samples"],
+                    inequality_constraints=inequality_constraints,
+                    fixed_features=fixed_features,
+                    X_pareto=self.acqf.X_baseline,
```

Comment:
We still need to update this
```suggestion
                    X_pareto=assert_is_instance(self.acqf.X_baseline, Tensor),
```